source/LoadPanel.cpp

```diff
@@ -366,6 +366,13 @@ bool LoadPanel::Scroll(double dx, double dy)
 
 
 
+bool LoadPanel::AllowFastForward() const
+{
+	return true;
```

Comment:
Why would anyone want to toggle fast-forward while being in the LoadPanel?

Reply:
They wouldn't, and it'll catch the keys when you're trying to type save file names. I've disabled it in the loading screen. Good catch.